backend/app/controllers/schedules_controller.ts

```diff
@@ -133,6 +134,12 @@ export default class SchedulesController {
       }
     }
 
+    if (payload.updatedAt) {
+      currSchedule.updatedAt = DateTime.fromJSDate(payload.updatedAt)
+    } else {
+      currSchedule.updatedAt = DateTime.now()
+    }
+
```

Comment:
autoupdate nie dziala?

Reply:
W modelach mamy takie coś, np. dla schedules: `@column.dateTime({ autoCreate: true, autoUpdate: true })
  declare updatedAt: DateTime`

Reply:
umm to nie dziala, i mimo ze najpierw tworze plan a pozniej go zaktualizowalem o dane
![image](https://github.com/user-attachments/assets/ca715d39-a360-41a8-bfdd-749977ce6625)
w kazdym razie i tak chce miec mozliwosc recznego updateuat, zeby miec ten sam na kliencie i backendzie bo na backendzie sila rzeczy automaty bedzie troche pozniej a potrzebuje ten sam

Reply:
plus moge wywalic tego else, i zostawic tylko jesli chce zaktualziowac

Reply:
okej, jak nazwe zmieniasz to sie tylko aktualizuje cnie, jak grupy, rejestracje czy kursy nie, teraz czaje

Reply:
> plus moge wywalic tego else, i zostawic tylko jesli chce zaktualziowac

tak zrób

Reply:
i mozesz mergowac